buildbot.py

```diff
@@ -42,11 +42,11 @@
     'x86_64-linux-debian8',
     'armv6-android-api9',
     'universal-mac-macosx10.9', # Minimum deployment target
-    'universal-ios-iphoneos11.3',
+    'universal-ios-iphoneos11.4',
```

Comment:
@livecodepanos `prebuilts-buildbot.py` needs this patch also. Now that we no longer maintain 8 it would be good to share the list of build triples in a common file.